evap/staff/tests/test_importers.py

```diff
@@ -181,34 +182,54 @@ def setUpTestData(cls):
         baker.make(CourseType, name_de="Vorlesung", import_names=["Vorlesung", "V"])
         Degree.objects.filter(name_de="Bachelor").update(import_names=["Bachelor", "B. Sc."])
         Degree.objects.filter(name_de="Master").update(import_names=["Master", "M. Sc."])
+        cls.default_excel_content = excel_data.create_memory_excel_file(excel_data.test_enrollment_data_filedata)
+        cls.create_existing_course(cls)
```

Comment:
You can just make `create_existing_course` a `@classmethod` and use `cls.create_existing_course()`

Reply:
(also, I'd argue that the merge for existing courses is a special case, not a base case. Thus I'd say `cls.create_existing_course()` should not be called in `setUpTestData`, but just in tests that need it)